low-level/message_handlers/node_data_msg_handler.py

```diff
@@ -412,6 +412,7 @@ def _generate_host_update(self):
                 # Transmit it out over rabbitMQ channel
                 self.host_sensor_data = jsonMsg
                 self.os_sensor_type["system"] = self.host_sensor_data
+                logger.info(f"_generate_host_update, jsonMsg: {jsonMsg}")

```

Comment:
prepend generic text and then further specific to alert, but not function name
"Raising alert for high memory usage: {jsonMsg}"

same comment for all other alerts

Reply:
Hold on. Addition of extra text will extend the message again & also it would need more changes 
So lets add small text for quick identification.

One suggestion to prepend like RAIL (stands for RAise ALert) per alert message & we wont differentiate between fault & faul_resolved alert.

Reply:
done.

Reply:
@shriya-deshmukh @cdeshmukh  Do we need this PR still?

Reply:
saurabh suggested to keep the PR only for main branch. Created the PR for main branch: https://github.com/Seagate/cortx-monitor/pull/212.  @saurabhkhanvilkar do I need to close this PR.
please let me know.

Reply:
Ideally wanted this in cortx-1.0, but now too late. 
>we need to merge this soon in cortx-1.0, not sure why this lagged behind ?

11th Nov had asked for this, not sure what made us wait still?


@saurabhkhanvilkar any window you see to push this in?

If not, will keep for main only.